### DISCUSSION TITLE
自定义BaseLayout

### DISCUSSION THREAD

**Discussion by lq9958** [vbenjs/vue-vben-admin#4403] [Q&A]
没玩儿过这种仓库管理模式，想自定义一下logo，发现无从下手，项目的配置状态也不知道从哪儿获取和修改，有知道的大佬可以指定一下吗？

**Comment by zhangyue-mars** [comment]
同问，改不动logo

**Comment by xingxingdcf** [comment]
更改 apps 所有 logo, packages => preferences,更改某个项目 logo,项目src => preferences.ts, 修改 logo.source 属性,其他配置修改同理

**Comment by dingdayu** [comment]
https://doc.vben.pro/guide/essentials/settings.html

文档里有，不过我有时还是有 copy Layout 的需求，通过 提PR  solt 支持太慢了
